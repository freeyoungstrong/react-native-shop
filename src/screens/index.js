export * from './login/login';
export * from './main/main';
export * from './productDetails/productDetails';
export * from './registration/registration';
export * from './welcome/welcome';
export * from './map/map';
