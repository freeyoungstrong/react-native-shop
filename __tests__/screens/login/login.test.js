import React from 'react';
import { shallow } from 'enzyme';

import { LoginScreen } from 'screens';

const mockNavigation = {
    navigate: jest.fn(),
};

describe('Login screen', () => {
    let wrapper;

    beforeEach(() => {
        wrapper = shallow(<LoginScreen navigation={mockNavigation} />);
    });

    it('should render fine', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
