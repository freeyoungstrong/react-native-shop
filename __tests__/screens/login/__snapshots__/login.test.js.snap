// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Login screen should render fine 1`] = `undefined`;
